modules/cnf-assigning-a-secondary-network-to-a-vrf.adoc

```diff
@@ -0,0 +1,128 @@
+[id="cnf-assigning-a-secondary-network-to-a-vrf_{context}"]
+= Assigning a secondary network to a VRF
+
+[IMPORTANT]
+====
+The VRF feature described here is for Tech Preview purposes only and is not supported by Red Hat at this time.
```

Comment:
You could include the technology-preview.adoc here instead of this note to keep consistent with other Tech Preview supportability statements.